# LiveTracks Message

std_msgs/Header header

std_msgs/Int32 trackId
std_msgs/Float32 dRel
std_msgs/Float32 yRel
std_msgs/Float32 vRel
std_msgs/Float32 aRel
std_msgs/Float32 timeStamp
std_msgs/Float32 status
std_msgs/Float32 currentTime
std_msgs/Bool stationary
std_msgs/Bool oncoming



# struct LiveTracks {
#   trackId @0 :Int32;
#   dRel @1 :Float32;
#   yRel @2 :Float32;
#   vRel @3 :Float32;
#   aRel @4 :Float32;
#   timeStamp @5 :Float32;
#   status @6 :Float32;
#   currentTime @7 :Float32;
#   stationary @8 :Bool;
#   oncoming @9 :Bool;
# }
